src/test/java/edu/wisc/my/messages/data/MessagesFromTextFileTest.java

```diff
@@ -4,13 +4,44 @@
 import static org.mockito.Mockito.when;
 
 import java.io.IOException;
+
+import org.junit.Assert;
 import org.junit.Test;
+
+import org.springframework.mock.env.MockEnvironment;
+
+import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.core.env.Environment;
+import org.springframework.core.io.DefaultResourceLoader;
 import org.springframework.core.io.Resource;
 import org.springframework.core.io.ResourceLoader;
 
 public class MessagesFromTextFileTest {
+  
+  /**
+   * Test that the data in the incoming text file is actually valid.
```

Comment:
Maybe this test should be in an integration (rather than unit) test? That is, it's not really so much about `MessagesFromTextFile` as it's about `messages.json`, which presumably we care about in the context of needing to use that file to drive other (integration) tests?